conf.js

```diff
@@ -0,0 +1,72 @@
+exports.hubURL = "https://hub.browserstack.com/wd/hub"
+
+exports.singleTestCapabilities = {
+  'bstack:options': {
+    "os": "OS X",
+    "osVersion": "Sierra",
+    "buildName": "Final-Snippet-Test",
+    "sessionName": "Selenium-4 Nodejs snippet test",
+    "local": "false",
+    "seleniumVersion": "4.0.0",
+    "userName": process.env["BROWSERSTACK_USER_NAME"] || "YOUR_USER_NAME",
+    "accessKey": process.env["BROWSERSTACK_ACCESS_KEY"] || "YOUR_ACCESS_KEY",
```

Comment:
```suggestion
    "userName": process.env.BROWSERSTACK_USERNAME || "YOUR_USER_NAME",
    "accessKey": process.env.BROWSERSTACK_ACCESS_KEY || "YOUR_ACCESS_KEY",
```

Reply:
Let's have userName and Access key at the top and then spread them in other capabilities similar to selenium3 conf file